src/components/SideBar.jsx

```diff
@@ -0,0 +1,97 @@
+// import React, { useState } from 'react';
+import {
+  FaTh,
+  FaUserAlt,
+  FaSignInAlt,
+} from 'react-icons/fa';
+import { NavLink } from 'react-router-dom';
+import Styles from './sidebar.module.css';
+
+const Sidebar = () => {
+  const menuItem = [
+    {
+      path: '/',
+      name: 'Home',
+      icon: <FaTh />,
+      end: true,
+
+    },
+    {
+      path: '/register',
+      name: 'Register',
+      icon: <FaUserAlt />,
+      end: false,
+      logged_in: false,
+    },
+    {
+      path: '/login',
+      name: 'Login',
+      icon: <FaSignInAlt />,
+      end: false,
+      logged_in: false,
+    },
+  ];
+  return (
+    <>
+      <div className="collapse d-md-none" id="navbarToggleExternalContent">
+        <div className="bg-light p-2">
+          <ul className="nav nav-pills flex-column mb-auto">
+
+            {menuItem.map((item, index) => (
+              <li key={index} className="nav-item text-dark text-decoration-none">
+                <NavLink
+                  to={item.path}
+                  className="nav-link text-decoration-none text-dark d-flex gap-3"
+                  end={item.end}
+                >
+                  <div className={Styles.icon}>{item.icon}</div>
+                  <div
+                    className={Styles.link_text}
+                  >
+                    {item.name}
+                  </div>
+                </NavLink>
+              </li>
+            ))}
+          </ul>
+        </div>
+      </div>
+      <nav className="navbar navbar-light bg-light d-md-none">
+        <div className="container-fluid">
+          <button className="navbar-toggler shadow-none" type="button" data-bs-toggle="collapse" data-bs-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span className="navbar-toggler-icon" />
+          </button>
+        </div>
+      </nav>
+
+      <div className="d-flex border-end flex-column flex-shrink-0 p-3 d-md-block d-none shadow-sm  rounded sidebar" style={{ width: '280px' }}>
+        <a href="sqs" className="d-flex align-items-center mb-3 mb-md-0 me-md-auto text-white text-decoration-none">
+          <img src="/images/logo.png" style={{ width: '240px', height: '170px' }} alt="" srcSet="" />
```

Comment:
[Optional] Don't you think it's better to create a new CSS class for the sizing of this image? Because inline CSS doesn't look quite professional.